app/__init__.py

```diff
@@ -0,0 +1,45 @@
+from flask import request, url_for
+from flask_api import FlaskAPI
+from . request_lookup import query_country_index, query_occupation_index, query_industry_index
```

Comment:
what does the `.` do in the from?

Reply:
That's the syntax for explicit relative imports. It means import from the current package. I'll just use app.request_lookup instead.